pre_commit_hooks/mixed_line_ending.py

```diff
@@ -0,0 +1,250 @@
+import argparse
+import logging
+import os
+import re
+import sys
+
+from enum import Enum
```

Comment:
Also, how can I install dependencies provided in `extras_require`? So far, it worked because I installed `enum34` with `sudo apt install python-enum34`, which is not the universal way to do this FWIU.